Makefile

```diff
@@ -18,3 +18,44 @@ wasm:
 
 parser:
 	go generate .
+
+build:
+	go build -ldflags=$(LDFLAGS) ./cmd/arrai
+
+
+######## Build information report
```

Comment:
The logic for collecting all this info seems a little shaky. Is it possible to just copy all the version stuff from https://github.com/anz-bank/sysl/blob/master/Makefile?

Reply:
It is very different from Sysl's implementation, for example:
```
$ arrai v
arrai 751a975944edadf56a2558a8a21eebcfd47fc52e~3 = v0.123.0 darwin/amd64
```
And `~3 = v0.123.0` indicates that v0.123.0 is three commits behind HEAD.
#246 lists all cases, and you can see all of them are different from Sysl.